### ISSUE TITLE
Xenia not working when unRAR-ed?

### ISSUE THREAD

**Issue by claude13112001** [benvanik/xenia#703]
- What's going wrong: Basically I download latest builds from here: https://ci.appveyor.com/project/benvanik/xenia (I think/hope it's an official source...) and when I run the Xenia executable file it launches normally (except for some antivirus software stuff popping up sometimes when I launch them first time but I ignore that cause I trust you guys :D ) but when I unpack it,it will only show the spinning wheel next to the cursor for a few moments and no interface will pop-up but it is somehow running for a few moments cause I can see it in the Task Manager before it disappears a little later.

- What should happen: Xenia should also launch normally when unpacked cause I've seen it in many videos...

- No logs or anything as Xenia won't even launch when unpacked. I know it's just a small problem pretty much but it still is weird.

My specs,in case that makes a difference:

Storage: 1TB HDD + 128GB SSD
OS: Windows 10 64-bit (installed on the SSD)
CPU: Intel Core i5 7400 Kaby Lake (3.2 GHz and 4 cores if I remember correctly) EDIT: 3.0 GHz actually :D
GPU: MSI GeForce GTX 1060 Armor 3GB
RAM: 16GB

Everything other should be compatible,installed etc. as Xenia can run AND also boot games normally when packed in a rar file...

I hope I am posting this in the right place.

**Comment by tmccaff** [comment]
Why don't you get the zip file; I always do with every commit. Go to https://ci.appveyor.com/project/benvanik/xenia/build/artifacts than hit JOBS > Configuration: Release look for tab Artifacts and than download that zip file that should be the newest commit.

**Comment by tmccaff** [comment]
Also disable antivirus when running

**Comment by claude13112001** [comment]
I get it,I am doing it like that,I just thought that it is a .rar file cause I wasn't (and am not now neither) at my PC so maybe it is a .zip file but whatever. I also add Xenia to the antivirus exceptions every time I get a new build (actually it quarantines it and then I recover it from there and it ignores it after that :D ) But I saw guys in videos running it normally even when they unpacked it so I don't know what's wrong...

**Comment by Margen67** [comment]
What antivirus are you using?

**Comment by claude13112001** [comment]
Is it that much of a problem when it runs zipped but not unzipped? But I have Kaspersky and Norton "trial versions" which were here already when I got this PC (well only Kaspersky,Norton showed up with some kind of a system update a few days ago) like a week or so ago if that really matters...

**Comment by claude13112001** [comment]
I also cannot change to OpenGL backend because I cannot make shortcuts of an .exe which is zipped :/ (according to this video: https://www.youtube.com/watch?v=z1pdgp8hd20 )

**Comment by hlide** [comment]
hem... you appear like a beginner  with your Windows.

1) it doesn't make a sense to try to run an executable from a .zip folder, you'd better unzip all the content inside a real folder. It may be okay for a standalone executable with no external data or executable (those also inside the zip) but probably not for the one which needs access them: with which absolute path do you think they would be accessible?  

2) why are you still using those kind of anti-virus softwares? they are bloatwares which tend to be in your way. Just use the one with Microsoft and watch your internet access (don't download and browse randomly) because the 0% risk never exists and fully leaving it up to anti-virus software is a retarded thing to do.

**Comment by claude13112001** [comment]
You don't understand -_- I tried unzipping it but it won't launch like that,it only launches when I start it from inside the zip file for some weird reason... Also,I am kind of a beginner with Windows 10 as I had only used it in school until now...

**Comment by Razzile** [comment]
that sounds like a bug with your particular PC. it's definitely not a xenia bug

**Comment by hlide** [comment]
So deactivate all your anti-virus bloatwares before unzipping it. By the way, what kind of unzipper software are you using? the internal one from Windows? Winrar? 7zip? try with another one when one is not working. And again try it with no anti-virus softwares in the way. We should really not answer you as it is clearly not an issue with Xenia.

**Comment by claude13112001** [comment]
I use WinRAR...

**Comment by claude13112001** [comment]
So I tried disabling all the antivirus software and unpack Xenia but ended up with the same result :/

**Comment by gibbed** [comment]
If you're using Windows 10 you probably need to open the properties of the .exe and unblock the file.

**Comment by claude13112001** [comment]
How can I do that?

**Comment by claude13112001** [comment]
Oh,you literally wrote it there but whatever XD

**Comment by claude13112001** [comment]
So I checked but there is no "unblock" checkbox in the Xenia exe properties...

**Comment by DrChat** [comment]
Hmm - that's odd behavior. When xenia is extracted, does it create a "xenia.log" in the same folder?

**Comment by paulsapps** [comment]
probably missing the redist or the AV is silently blocking by detecting it as a FP

**Comment by DrChat** [comment]
Both of those things should be equally as likely to happen when running Xenia from the compressed folder.

**Comment by claude13112001** [comment]
Nope,it does not create a "xenia.log" file when extracted... :/

**Comment by claude13112001** [comment]
What's up with all those thumbs down? I am just asking a normal question so stop it... -_-

**Comment by hlide** [comment]
First, I must recall you that issue tracker is for development issues, not for user issue, hence those "thumbs downs". The issue you're describing is not a bug to a code but due to your personal PC behavior, and so should *not* be discussed here. 

To be sure, I downloaded the last artefact from Appveyor and extracted it with 7-zip: I was perfectily able to launch the executable and get the log file as well. So nobody can help you here since it is now clear the faulty one is not the code or Appveyor.

**Comment by claude13112001** [comment]
Then maybe you could tell me rather than putting those damn thumbs down?

**Comment by claude13112001** [comment]
And also,where else should I post this?

**Comment by hlide** [comment]
Did I put "those damn thumbs down"? surely not. Can I help you? Why? If you could understand why we cannot help you (hint: your PC behavior), you wouldn't ever ask that. Your trouble is only yours and none of us have any clue about why your "damn PC" doesn't want to extract Xenia perfectly. So be smarter and handle it yourself.

**Comment by claude13112001** [comment]
I didn't say you put it there. I also didn't write the application so I just thought that (as you know the damn code of the thing and can orientate in it as I don't know C# or whatever you write it in) you could help me with that but ok whatever,I'll just deal with launching it from the .zip file and hope it'll somehow magically fix in the newer commits. Just close this,we're not going to find a solution with this atittude anyway...

**Comment by DrChat** [comment]
@claude13112001 Where are you extracting Xenia?

**Comment by claude13112001** [comment]
D:\My Folder (it has a different name actually)\Xenia Xbox 360 Emulator 
(D:/ is the "second" disk in my PC,the 1TB HDD one which doesn't have Windows installed on it)

**Comment by DrChat** [comment]
Try running it as an administrator and seeing what happens.

**Comment by claude13112001** [comment]
Still the same,with the difference that a "Did this program work correctly?" window also pops-up...

**Comment by hlide** [comment]
Just try with another one, say: https://ci.appveyor.com/project/rpcs3/rpcs3/branch/master/artifacts

**Comment by claude13112001** [comment]
I don't know if you actually meant to post a link to RPCS3 but I tried it anyway and it's working normally,even when unpacked.

**Comment by hlide** [comment]
no, I just mean you can try other projects having an artefact in the same way to check. RPCS3 appeared to have one but nothing prevents from you to check another one. I don't have any issue with Xenia and I'm pretty sure it is the case for other people here. Now, if you told there is no issue with another one, it seems you had messed up with your PC (I suppose you tried several unzipper softwares - if not, try with 7-zip - as it worked with mine and maybe your WinRAR is corrupting your extraction). Except for removing your anti-virus softwares completely to be sure they are not in the way, I see no real solution for you. You'd probably better to google it in a more general way, not specifically for Xenia as you may be the only unlucky guy here.

**Comment by claude13112001** [comment]
Okay,I'll just keep trying and hope I possibly find a solution. Thanks for the help and sorry for me being so rude earlier,I don't really know how it works on this Issue Trackers and stuff...

**Comment by claude13112001** [comment]
So I just checked and found this in the Event Viewer,I think it seems kinda interesting and might maybe point to the solution after all but I think you'll be able to read it more than I will,I put it in a file so I won't make a mess out of it here...
[Xenia Event Viewer errors.txt](https://github.com/benvanik/xenia/files/999679/Xenia.Event.Viewer.errors.txt)

**Comment by gibbed** [comment]
`0xC0000409` would imply the problem is likely with your system/Windows and not Xenia.

**Comment by simontoft** [comment]
I recreated the issue, xenia doesn't seem to launch when the path of the exe includes non-ascii characters:

Doesn't work:
C:\é\xenia\xenia.exe
C:\å\xenia\xenia.exe
C:\¥\xenia\xenia.exe

Works:

C:\a\xenia\xenia.exe
C:\#\xenia\xenia.exe
C:\!\xenia\xenia.exe

**Comment by claude13112001** [comment]
What can I possibly do to fix it?

**Comment by claude13112001** [comment]
Oh,I get it now! Thanks,will try it when I get to my PC tommorow...

**Comment by simontoft** [comment]
@claude13112001 - Extract xenia to a path that does not include a non-ascii character. It works from the zip/rar, because it extracts them to c:\temp when ran.

**Comment by gibbed** [comment]
@simontoft Thank you. I'll look into this.

**Comment by hlide** [comment]
good catch @simontoft.

**Comment by claude13112001** [comment]
So we actually found out that it IS a "bug" of some sort in Xenia,I hope the logs were at least a little useful,although I admit that we maybe could've reached this conclusion earlier if I didn't "censore out" my folders name earlier. So let me be a little rude one more time,take those /damn/ "thumbs down" and shove'em up your... ;) (only referring to the ones who put those "thumbs down" there...)

**Comment by claude13112001** [comment]
I'll try it out tommorownand report back with the results as I am on my mobile now and don't have access to my computer at the moment (it's late night here already).

**Comment by claude13112001** [comment]
Ok,can confirm that it now works,even in a folder that contains non-ASCII characters in it's name...

**Comment by udance4ever** [comment]
> I recreated the issue, xenia doesn't seem to launch when the path of the exe includes non-
characters:
> 
> Doesn't work: C:\é\xenia\xenia.exe C:\å\xenia\xenia.exe C:\¥\xenia\xenia.exe
> 
> Works:
> 
> C:\a\xenia\xenia.exe C:\#\xenia\xenia.exe C:\!\xenia\xenia.exe

thank you for this tip.  Turns out even spaces didn't work:
```
C:\Program Files\Xenia\xenia_canary.exe
```
I moved it here and it works:
```
S:\system\configs\xenia\xenia_canary.exe
```
Only difference I see is no spaces!
